tools/shared/src/main/scala/org/scalajs/core/tools/jsdep/JSDependencyManifest.scala

```diff
@@ -87,30 +87,57 @@ object JSDependencyManifest {
    *  resourceName -> ResolutionInfo
    */
   private def mergeManifests(flatDeps: Traversable[FlatJSDependency]) = {
-    @inline
-    def hasConflict(x: FlatJSDependency, y: FlatJSDependency) = (
-      x.commonJSName.isDefined &&
-      y.commonJSName.isDefined &&
-      (x.relPath == y.relPath ^
-          x.commonJSName == y.commonJSName)
-    )
+    checkCommonJSNameConflicts(flatDeps)
 
-    val conflicts = flatDeps.filter(x =>
-      flatDeps.exists(y => hasConflict(x,y)))
+    val byRelPath = flatDeps.groupBy(_.relPath)
 
-    if (conflicts.nonEmpty)
-      throw new ConflictingNameException(conflicts.toList)
+    checkMinifiedJSConflicts(byRelPath)
 
-    flatDeps.groupBy(_.relPath).mapValues { sameName =>
+    byRelPath.mapValues { sameName =>
       new ResolutionInfo(
         relPath = sameName.head.relPath,
         dependencies = sameName.flatMap(_.dependencies).toSet,
         origins = sameName.map(_.origin).toList,
-        commonJSName = sameName.flatMap(_.commonJSName).headOption
+        commonJSName = sameName.flatMap(_.commonJSName).headOption,
+        relPathMinified = sameName.flatMap(_.relPathMinified).headOption
       )
     }
   }
 
+  private def checkCommonJSNameConflicts(flatDeps: Traversable[FlatJSDependency]) = {
+    @inline
+    def hasConflict(x: FlatJSDependency, y: FlatJSDependency) = (
+        x.commonJSName.isDefined &&
+        y.commonJSName.isDefined &&
+        (x.relPath == y.relPath ^ x.commonJSName == y.commonJSName)
+    )
+
+    val conflicts = flatDeps.filter(x =>
+      flatDeps.exists(y => hasConflict(x, y)))
```

Comment:
With `()`, the `x =>` should be on the next, together with the body lambda. Then, the second line should be indented with 4 spaces instead of 2.
(I know it's a cut-paste from before, but Boy Scout Rule ^^)